src/legacy/core_plugins/metrics/public/visualizations/components/_top_n.scss

```diff
@@ -11,6 +11,16 @@
   }
 }
 
+.tvbVisTopN__table {
+  width: 100%;
+}
+
+.tvbVisTopN__table tr > td:nth-child(1) {
```

Comment:
Isn't this just selector just `.tvbVisTopN__label`?

Reply:
Yep, you are completely right! Just a messy code since trying various techniques of implementation